supadata/youtube.py

```
@@ -1,7 +1,7 @@
 """YouTube-related operations for Supadata."""
 
 from datetime import datetime
-from typing import Any, Callable, Dict, List
+from typing import Any, Callable, Dict, List, Optional, Literal
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)